src/regtech_data_validator/create_schemas.py

```diff
@@ -177,6 +177,10 @@ def validate_phases(df: pd.DataFrame, context: dict[str, str] | None = None) ->
     p1_is_valid, p1_findings = validate(get_phase_1_schema_for_lei(context), df)
 
     if not p1_is_valid:
+        p1_findings.insert(1, "validation_phase", PHASE_1_TYPE, True)
         return p1_is_valid, p1_findings
```

Comment:
I'm wondering if we could add the type in the return tuple as well; I was originally thinking of just adding it to the return tuple without modifying the findings object; and the downstream API can use the type to do with it what it wants.

Reply:
oh right... insert does column insert... not row... nvm about not modifying the findings object

Reply:
We could do this instead.  I realized I need to at least set something either in the json, or through this approach, so the filing-api will know what to send in the submission object metadata.  Your call.  But yeah this is just adding a new column to the result set.  Only reason this approach may be desirable is if (and I know for MVP it's not a thing) we ever end up with a mixture of syntactical and logical errors going back, we'd want a designator on each finding, not just the return call.

Reply:
yeap, I don't see the harm in having both, I can see utility in the downstream call to have one more thing in the return, and the column addition is good for the csv.

Reply:
Agreed. I like having the phase in the return tuple. Nice to be able to drive logic without having to look into the DataFrame.

Adding it to the DataFrame seems fine too. I don't think it hurts anything, unless we need to cut things later to optimize things for size or performance.